src/main/java/com/epam/msu/dao/SqlRequest.java

```diff
@@ -0,0 +1,26 @@
+package com.epam.msu.dao;
+
+public final class SqlRequest {
+
+    private SqlRequest() {
+    }
+
+    public static final String getAllCertificates = "SELECT * FROM certificate";
```

Comment:
if it static final - then GET_ALL_SERTIFICATES (and please fix all such namings)